Spec/RealtimeClientPresence.swift

```diff
@@ -426,25 +426,28 @@ class RealtimeClientPresence: QuickSpec {
                     defer { client2.close() }
                     let channel2 = client2.channels.get("test")
 
+                    var count = 0
+                    channel1.presence.subscribe(.Update) { member in
+                        expect(member.action).to(equal(ARTPresenceAction.Update))
+                        expect(member.clientId).to(equal("john"))
+                        expect(member.data as? NSObject).to(equal("away"))
+                        count += 1
+                    }
+
                     waitUntil(timeout: testTimeout) { done in
-                        let partlyDone = AblyTests.splitDone(2, done: done)
-                        channel1.presence.subscribe(.Update) { member in
-                            expect(member.action).to(equal(ARTPresenceAction.Update))
-                            expect(member.clientId).to(equal("john"))
-                            expect(member.data as? NSObject).to(equal("away"))
-                            partlyDone()
```

Comment:
Honestly, I don't know. I spend so much time with this: the flow is correct, the log doesn't show anything weird, I tried debug the problem but it never happened when I was debugging.